Sources/MapboxNavigation/RouteLineController.swift

```diff
@@ -47,8 +47,8 @@ extension NavigationMapView {
         // MARK: - Private methods
         
         private func showRouteIfNeeded() {
-            guard navigationViewData.containerViewController.isViewLoaded &&
-                    navigationViewData.containerViewController.view.window != nil else { return }
```

Comment:
@Udumft, would you be able to provide more details on why you've added this check: `navigationViewData.containerViewController.view.window != nil`? At this point it prevents from showing route line right at the start of active guidance navigation session, when using `NavigationViewController`. I wonder if I miss anything here.